test/EnvironmentManagerTests/Helper.h

```diff
@@ -0,0 +1,167 @@
+﻿#pragma once
```

Comment:
'Helper' is so unhelpfully <g> generic.

Please choose a better name

And if there's multiple things going on here split into multiple functionally-oriented files. Better that than one large mosh pit of a dumping ground